packages/nc-gui/components/project/spreadsheet/components/columnFilter.vue

```diff
@@ -157,26 +170,20 @@
       </template>
     </div>
 
-    <v-btn
-      small
-      class="elevation-0 grey--text my-3"
-      @click.stop="addFilter"
-    >
-      <v-icon small color="grey">
-        mdi-plus
-      </v-icon>
+    <v-btn small class="elevation-0 grey--text my-3" @click.stop="addFilter">
+      <v-icon small color="grey"> mdi-plus </v-icon>
       <!-- Add Filter -->
-      {{ $t('activity.addFilter') }}
+      {{ $t("activity.addFilter") }}
     </v-btn>
     <slot />
   </div>
 </template>
 
 <script>
-import { UITypes } from '~/components/project/spreadsheet/helpers/uiTypes'
+import { UITypes } from "~/components/project/spreadsheet/helpers/uiTypes";
```

Comment:
i think all double quotes should be reverted back to single quotes because javascript standard is single quotes

Reply:
Hi @dammy001 ,thanks for point out, I will make sure, I should not add any extra formatting for the commits